tests/std/tests/P0553R4_bit_rotating_and_counting_functions/test.cpp

```diff
@@ -105,6 +105,27 @@ constexpr bool test_rotr() {
     return true;
 }
 
+// tests functions for 64bit operands that
+// have either high or low halves as zero
+// these may be split into two operations
+// on 32bit platforms and we need to check
+// if we handle the == zero or == ones cases
+// correctly

```

Comment:
I believe that these are complete sentences which should have proper capitalization and punctuation.